librz/reg/rvalue.c

```diff
@@ -229,6 +229,20 @@ RZ_API bool rz_reg_set_value(RzReg *reg, RzRegItem *item, ut64 value) {
 	case 8:
 		rz_write_ble8(src, (ut8)(value & UT8_MAX));
 		break;
+	case 4:;
```

Comment:
A typo

Reply:
The next line is a statement but after a `case` must follow an expression. The single `;` is there as a single expression. Otherwise it will not compile.

Reply:
A better way is to just add `{` and `}` instead, no?

Reply:
Yeah, you are right. Looks indeed better.